core/src/main/java/com/google/instrumentation/stats/Stats.java

```diff
@@ -26,7 +26,7 @@
    * Returns the default {@link StatsContextFactory}.
    */
   public static StatsContextFactory getStatsContextFactory() {
-    return statsManager.getStatsContextFactory();
+    return statsManager == null ? null : statsManager.getStatsContextFactory();
```

Comment:
Create a separate issue for this: https://github.com/google/instrumentation-java/issues/215